frontend/src/components/Search.tsx

```diff
@@ -0,0 +1,21 @@
+import clsx from "clsx";
+import { TextField } from ".";
+
+export interface SearchProps {
```

Comment:
This will need `value` and `onChange` props that are forwarded to the text field so that it's value can be tracked.
Components should also be forwarding refs, but that's a separate issue that I'll deal with en-mass later.